### PR TITLE
Enable Email Sending

### PR DESCRIPTION
**Pull request by ahsan-ul-haq**
ECOM-2078

@zubair-arbi @clintonb @feanil please review

### CODE REVIEW

File: ecommerce/extensions/checkout/signals.py [hunk 1]
```diff
@@ -62,31 +62,30 @@ def track_completed_order(sender, order=None, **kwargs):  # pylint: disable=unus
 @log_exceptions("Failed to send order completion email.")
 def send_course_purchase_email(sender, order=None, **kwargs):  # pylint: disable=unused-argument
     """Send course purchase notification email when a course is purchased."""
-    if waffle.switch_is_active('ENABLE_NOTIFICATIONS'):
```
Comment (clintonb) [inline ecommerce/extensions/checkout/signals.py:65]: What was this removed? During grooming we stated that no _new_ switches would be added. The existing switch should remain.

File: requirements/base.txt [hunk 2]
```diff
@@ -1,10 +1,12 @@
 analytics-python==1.0.3
+boto==2.38.0
```
Comment (clintonb) [inline requirements/base.txt]: Unless you are calling `boto` directly, it should not be listed here. `pip` will handle finding the `django-ses` dependencies.

File: ecommerce/settings/local.py [hunk 3]
```diff
@@ -17,8 +17,7 @@
 # END DEBUG CONFIGURATION
 
 # EMAIL CONFIGURATION
-# See: https://docs.djangoproject.com/en/dev/ref/settings/#email-backend
-EMAIL_BACKEND = 'django.core.mail.backends.console.EmailBackend'
+EMAIL_BACKEND = 'django_ses.SESBackend'
```
Comment (clintonb) [inline ecommerce/settings/local.py]: SES is not needed locally. It's needed for production.

File: requirements/base.txt [hunk 4]
```diff
@@ -5,6 +5,7 @@ django-compressor==1.5
 django_extensions==1.5.5
 django-libsass==0.3
 django-oscar==1.1
+django-ses==0.7.0
```
Comment (feanil) [inline requirements/base.txt]: Shouldn't this go into the production requirements.txt file at `requirements/production.txt`?

**Comment by ahsan-ul-haq** [comment]
@feanil @clintonb I have checked email sending locally by setting up django-ses. Following is the screen shot of email sent.

screen shot 2015-09-07 at 6 06 56 pm

**Comment by clintonb** [comment]
:+1:

Please coordinate with @feanil to ensure the sender's address (billing@edx.org) is verified and properly set in configuration. Nice work!

**Comment by feanil** [comment]
Is the sender e-mail address hard coded?  Can we pull it out into configuration in case we want to change it?

**Comment by clintonb** [comment]
The address is a setting that needs to be added to configuration.

## 

Clinton Blackburn

**Comment by ahsan-ul-haq** [comment]
@feanil Default sender email address needed to be added in the settings as `OSCAR_FROM_EMAIL`. Currently it used default value which is `oscar@example.com`. I wasn't sure which email address to be added so once you guys let me know I will add that as well.

FYI: @clintonb

**Comment by feanil** [comment]
@clintonb @ahsan-ul-haq That seems fine as a default, let's assume we'll override it in config_secure.  Please open a ticket for adding this change to configuration_secure.  Please mention that the e-mail addresses should be verified by SES as well.

**Comment by clintonb** [comment]
@ahsan-ul-haq https://github.com/ahsan-ul-haq please open a ticket,
requesting the address be set to billing@edx.org.

## 

Clinton Blackburn

**Comment by ahsan-ul-haq** [comment]
@clintonb @feanil I have opened a ticket [DEVOPS-2784](https://openedx.atlassian.net/browse/DEVOPS-2784). So should I merge this one?

**Comment by clintonb** [comment]
@ahsan-ul-haq merge away.

**Pull request merged**